uber/templates/groupform.html

```diff
@@ -103,14 +103,8 @@
         <p class="help-block col-sm-9 col-sm-offset-3">The one you want us to link on our website, or n/a</p>
     </div>
 
-    {% if not c.COLLECT_FULL_ADDRESS %}
-    <div class="form-group">
-        <label for="address" class="col-sm-3 control-label">Address and Phone Number</label>
-        <div class="col-sm-6">
-            <textarea class="form-control" name="address" rows="4" placeholder="Your or your business' full address and phone number for tax purposes.">{{ group.address }}</textarea>
-        </div>
-    </div>
-    {% endif %}
+    {% set address_form=True %}
```

Comment:
... whoops, that line is left over from when I tried to use Jinja2 variables to control `include` statements. I gotta get rid of it.